integrations/oap/kubernetes/docker/oap-centos-benchmark/Dockerfile

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 #
 
-FROM oap-centos:1.2.0
+FROM oap-ubuntu:1.3.0
 
```

Comment:
You can directly remove entire oap-centos-benchmark/Dockerfile.
Because oap-ubuntu-benchmark exists.

Reply:
done